observable-slim.js

```diff
@@ -161,6 +161,9 @@ var ObservableSlim = (function() {
 
 				// for performance improvements, we assign this to a variable so we do not have to lookup the property value again
 				var targetProp = target[property];
+			    if (target instanceof Date && targetProp instanceof Function && targetProp !== null) {
```

Comment:
There's something weird going on with the indenting and I'm not sure how the Travis CI lint check didn't catch it -- it's mixing tabs and spaces for indenting on the same line. If you review the code on GitHub you'll see there are 3 tabs followed by 4 spaces. Looking at your last PR, I think the same thing happened in the unit testing code too.